packages/destination-actions/src/destinations/fullstory/__tests__/fullstory.test.ts

```diff
@@ -109,10 +113,11 @@ describe('FullStory', () => {
         anonymousId,
         traits: {
           email,
-          displayName,
-          'originally-hyphenated': true,
+          name: displayName,
```

Comment:
This was another test bug that was exposed with type suffixing. This always should have been `name` to test the default mapping of `name => displayName`.